README.md

```diff
@@ -1 +1,23 @@
-# hw01
\ No newline at end of file
+# hw01
+# About Warren Casey Epstein
+## Undergraduate Student at the University of Chicago
+
+
+## Rising Third-Year in the College
+I am a rising third-year in the College, double majoring in **political science** and **economics** *with a specialization in business economics*. I will also hopefully minor in *near eastern languages and civilizations*.
```

Comment:
I like how you use emphasis to make a distinction between majors and minors. Very creative!